ReleaseHistory.md

```diff
@@ -1,5 +1,10 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 
+## **Unreleased**
+* DEP: Explicitly add `Azure.Identity` 1.10.2 in `Sarif.Multitool.Library` and `WorkItems` to fix the vulnerability in 1.3.0 via `Microsoft.Azure.Kusto.Data` 10.0.3.
```

Comment:
>the [](http://example.com/codeflow?start=98&length=3)

can we say 'a' vulnerability here as I suspect that the one in question is not the sole bug in 1.3.0.

Also should we explicitly say 'fix a security vulnerability' ?

Reply:
how about "... to avoid the vulnerable package per compliance requirements"?